crypto/tbs/src/lib.rs

```diff
@@ -185,6 +185,14 @@ fn lagrange_multipliers(scalars: Vec<Scalar>) -> Vec<Scalar> {
         .collect()
 }
 
+pub fn verify_blinded_signature(
```

Comment:
Some docs might be useful here for everything that isnt familiar with the pairing crypto